src/wallet/wallet.h

```diff
@@ -64,6 +64,10 @@ static const bool DEFAULT_WALLET_RBF = false;
 static const bool DEFAULT_WALLETBROADCAST = true;
 static const bool DEFAULT_DISABLE_WALLET = false;
 
+//! Pre-calculated constants for input size estimation in *virtual size*
+static constexpr size_t DUMMY_NESTED_P2WPKH_INPUT_SIZE = 91;
+static constexpr size_t DUMMY_NESTED_P2WPKH_INPUT_SIZE_MAX = 91;
```

Comment:
This constant isn't used anywhere, is that intentional?

Reply:
As of now ‘_MAX’ is used in wallet_tests... could maybe do without it (given equality), with further commenting.